### PR TITLE
Changed almost all instances of Oakland, I've commented stuff I'm not sure about. 

### PR DESCRIPTION
**Pull request by mayanhavoc**

### CODE REVIEW

File: _src/_archived/oak11-13adopted2.html [hunk 1]
```diff
@@ -122,7 +120,7 @@ <h1>
 	<div class="row">
 
 		<div class="ten columns">
-		<div><h4 class="subheader">FY 2011-13 City of Oakland Expenditures, Adopted</h4></div>
+		<div><h4 class="subheader">FY 2011-13 City of ### Expenditures, Adopted</h4></div>
```
Comment (feydan) [inline _src/_archived/oak11-13adopted2.html:123]: Should be `Sacramento` instead of `###`

File: _src/css/main.css [hunk 2]
```diff
@@ -0,0 +1,8248 @@
+@charset "UTF-8";
```
Comment (feydan) [inline _src/css/main.css:1]: Looks like you may have added this file?  perhaps it was created when building the app?

File: _src/css/main.css.map [hunk 3]
```diff
@@ -0,0 +1,88 @@
+{
```
Comment (feydan) [inline _src/css/main.css.map:1]: Looks like you may have added this file? perhaps it was created when building the app?

File: _src/dist/_layout.html [hunk 4]
```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <meta name="viewport" content="width=device-width">
-    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="../css/main.css">
```
Comment (feydan) [inline _src/dist/_layout.html:5]: Why did you change the path here?

File: _src/dist/_layout.html [hunk 5]
```diff
@@ -53,17 +53,17 @@
       <div class="container">
         <div class="row">
           <div class="col-md-6 col-sm-9">
-            <p>Open Budget: Oakland is an open-source project by<a href="http://openoakland.org">OpenOakland</a> to help citizens better understand Oakland's spending and budget process.</p>
+            <p>Open Budget: Sacramento is an open-source project by<a href="http://openbudgetsac.org/">Open Budget Sacramento</a> to help citizens better understand Sacramento's spending and budget process.</p>
             <ul class="list-inline">
-              <li><a href="http://openoakland.org"><img src="/images/global/openoakland-logo.png" width="150" alt="OpenOakland"></a></li>
-              <li><a href="https://twitter.com/openbudgetoak"><img src="/images/global/twitter-icon.png" alt="twitter"></a></li>
-              <li><a href="https://github.com/openoakland/openbudgetoakland"><img src="/images/global/github-icon.png" alt="github"></a></li>
+              <li><a href="http://openbudgetsac.org/"><img src="_src/images/global/codeforsac-logo.png" width="150" alt="Open Budget Sacramento"></a></li>
```
Comment (feydan) [inline _src/dist/_layout.html:58]: Why did you change the path here?

**Review by feydan — CHANGES_REQUESTED** [review]
Let's go through this together.  I think everything in `_src/dist` is generated programmatically.  I'm not sure why its checked into git in the first place.  We can take a look.

File: _src/dist/_layout.html (hunk 4 above)
Comment (feydan) [reply _src/dist/_layout.html:5]: actually may not be needed since it is in `_src/dist`, we can review together.

File: _src/dist/_layout.html (hunk 5 above)
Comment (feydan) [reply _src/dist/_layout.html:58]: actually may not be needed since it is in `_src/dist`, we can review together.

Comment (mayanhavoc) [reply _src/dist/_layout.html:58]: Ok, so I had a dumb issue with my repository. I had something like 5,000 staged commits, it was a mess and I might have messed a LOT of my directories trying to fix it. Could that be a reason why the paths are all messed up? It seems like the issue is repeating so I'm guessing it's an issue on my end.

File: README.md [hunk 6]
```diff
@@ -59,7 +59,7 @@ This project is coded with:
 ## Creating & Editing Pages
 
 - Please note that it is your responsibility to keep your fork of the repo up-to-date with changes made by others working on the project. Doing this diligently should go a long way towards protecting you from scary git merge conflicts.
-- All development activity occurs in `_src/`. The root folder is only for compiled output for deployment.
+- All development activity occurs in \_src/ The root folder is only for compiled output for deployment.
```
Comment (feydan) [inline README.md:62]: Revert this change.

File: _src/_archived/oak11-13adopted2.html [hunk 7]
```diff
@@ -108,9 +108,7 @@ <h1>
 		<li class="divider"></li>
                 <li><a href="feedback.html">Feedback</a></li>
 		<li class="divider"></li>
-                <li><a href="https://twitter.com/search/realtime?q=%23OAKbudget&src=typd">#OAKbudget</a></li>
-
-
+                <li><a href="https://twitter.com/search/realtime?q=%23OAKbudget&src=typd">###budget</a></li>
```
Comment (feydan) [inline _src/_archived/oak11-13adopted2.html:111]: remove this line item

File: _src/css/main.css (hunk 2 above)
Comment (feydan) [reply _src/css/main.css:1]: Remove this file

File: _src/css/main.css.map (hunk 3 above)
Comment (feydan) [reply _src/css/main.css.map:1]: Remove this file
